lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -139,7 +141,7 @@ function M.select_download_commands(repo, project_name, cache_folder, revision)
         opts = {
           args = {
             '-L', -- follow redirects
-            is_github and url.."/archive/"..revision..".tar.gz"
+            is_github_or_gitlab and url.."/archive/"..revision..".tar.gz"
```

Comment:
Have to change that back, because  we have to use the first url for github and the second for gitlab

Reply:
It crashes the Linux/MacCI
```
 Failed to execute the following command:
{
  cmd = "tar",
  err = "Error during tarball extraction.",
  info = "Extracting...",
  opts = {
    args = { "-xvf", "tree-sitter-jsonc.tar.gz", "-C", "tree-sitter-jsonc-tmp" },
    cwd = "/home/runner/.local/share/nvim"
  }
}
```

Reply:
Just as an additional note, this is ta's response:
`tar: Error opening archive: Unrecognized archive format`

Reply:
It's unrecognized because Gitlab will post html when the URL is invalid.